frontend/www/js/omegaup/components/login/Signup.test.ts

```diff
@@ -0,0 +1,41 @@
+import { shallowMount } from '@vue/test-utils';
+
+import login_Signup from './Signup.vue';
```

Comment:
You'll need use the following component:
```suggestion
import login_Signup from './Signup.vue';
import omegaup_DatePicker from '../DatePicker.vue';
```